airflow/api_connexion/openapi/v1.yaml

```diff
@@ -38,7 +38,8 @@ paths:
   /connections:
     get:
       summary: Get all connection entries
-      operationId: airflow.api_connexion.endpoints.connection_endpoint.get_connections
+      x-openapi-router-controller: airflow.api_connexion.endpoints.connection_endpoint
```

Comment:
@mik-laj I am thinking of pulling this openapi spec change into a separate PR so we can get it merged in early to avoid conflicts. what do you think?

Reply:
Good idea. This will reduce the number of conflicts.

Reply:
alright, sent #9495.